pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -582,11 +582,17 @@ def upload_folder_to_dandi(
     cleanup: Optional[bool] = None,
     number_of_jobs: Optional[int] = None,
     number_of_threads: Optional[int] = None,
+    ignore_cache: Optional[bool] = None,
```

Comment:
```suggestion
    ignore_cache: bool = False,
```

No big reason to have `None` in this case, right?  Since it maps from a checkbox, which is either marked or not